.github/workflows/ci.yaml

```diff
@@ -151,10 +189,11 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v3
         with:
-          name: gradle_cosmos_integration_test_reports
+          name: cosmos_integration_test_reports
```

Comment:
Since we have many test output artifacts, I simplified the file name to browse through them easily.